HotSpot/AppDelegate.m

```diff
@@ -7,24 +7,83 @@
 //
 
 #import "AppDelegate.h"
-#import "Parse/Parse.h"
+#import "DataManager.h"
+#import "Booking.h"
 
 @interface AppDelegate ()
 
 @end
 
 @implementation AppDelegate
 
+PFUser *homeowner;
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(  NSDictionary *)launchOptions {
     
-    ParseClientConfiguration *config = [ParseClientConfiguration   configurationWithBlock:^(id<ParseMutableClientConfiguration> configuration) {
-        
-        configuration.applicationId = @"myAppId";
-        configuration.server = @"http://hotspot2017.herokuapp.com/parse";
+    [DataManager configureParse];
+    
+    [DataManager getHomeownersWithCompletion:^(NSArray<PFUser *> * _Nonnull homeowners, NSError * _Nonnull error) {
+        NSLog(@"%@", homeowners);
+        homeowner = homeowners[0];
```

Comment:
are we guaranteed to only have one home owner? what if there are many? How are we handling this case?